src/app/demo/nav/nav.component.scss

```diff
@@ -0,0 +1 @@
+@import '../scss/bootstrap/bootstrap.min';
```

Comment:
This is a bootstrap nav bar, and we don't want to include bootstrap anywhere, only in the demo